### ISSUE TITLE
DLC is Not Working

### ISSUE THREAD

**Issue by Charles-Perez** [sudachi-emu/sudachi#137]
You first install the dlc and play the game and after you close sudachi and reopen it the dlc is not installed anymore

**Comment by carlosgamer23** [comment]
> Você primeiro instala o dlc e joga o jogo e depois de fechar o sudachi e reabri-lo o dlc não está mais instalado

First is the update, then the DLC, and in some cases they only recognize the DLC if you delete your old save and start a new one

**Comment by zzzribas** [comment]
Yes, im also getting this problem, if i install manually it works until i close the emulator and open again, i changed my directory of nand can it be it? Does it check when it starts for the default nand and after that unloads the addons or dlcs?

**Comment by MercAlert** [comment]
I'm also getting this exact issue.
